storage/src/main/java/tech/pegasys/teku/storage/server/StorageConfiguration.java

```diff
@@ -284,12 +287,17 @@ public Builder statePruningInterval(final Duration statePruningInterval) {
       if (statePruningInterval.isNegative() || statePruningInterval.isZero()) {
         throw new InvalidConfigurationException("Block pruning interval must be positive");
       }
+      if (statePruningInterval.toSeconds() < 30L
+          || statePruningInterval.toSeconds() > Duration.ofDays(1).toSeconds()) {
+        throw new InvalidConfigurationException(
+            "Block pruning interval must be a value between 30 seconds and 1 day");
+      }
```

Comment:
Are we missing unit tests for `StorageConfiguration` for the desired checks?

Reply:
Added some unit test just to make sure that we test those limits and sanity checks in the tests as well